chart/newsfragments/30054.improvement.rst

```diff
@@ -0,0 +1 @@
+The pgbouncer and pgbouncer-images are now based on newer software and os versions. Thye also have native AMD64 and ARM64 support (previously only AMD had native support)
```

Comment:
```suggestion
The pgbouncer and pgbouncer-images are now based on newer software and os versions. They also have native AMD64 and ARM64 support (previously only AMD had native support).
```

It might be worth doing a significant entry, so we can more easily list out what each one updated to.

Reply:
Done.